atsign-foundation/at_wavi_app
# Fixes Issues that were stopping us from deploying to Google Play store

**- What I did**

Fixed tiny issues that were throwing errors and warnings when we changed SDK version 33 to 34

**- How I did it**

Initially faced with 250 issues, using ```dart fix``` fixed more than 190 issues and I had to do minor changes with regards to deprecated syntax, logical discrepancies and removing unnecessary code

**- How to verify it**

Cross verify with the previous codebase to the current change for most of the fixes but also there is a requirement to check out latest Flutter documentation for few widgets and properties.

**- Description for the changelog**

Fixed issues that were stopping us from upgrading to sdk version 34 which is required to deploy in Google Play store
